lib/stamp/translator.rb

```diff
@@ -79,7 +78,7 @@ def time_emitter(token)
         Emitters::AmPm.new
 
       when MERIDIAN_UPPER_REGEXP
-        Emitters::AmPm.new { |v| v.upcase }
+        Emitters::AmPm.new(['AM','PM'])
```

Comment:
Then you really wouldn't have liked my alternative:

```ruby
Emitters::Lookup.new(:hour, %w(am) * 12 + %w(pm) * 12)
```
But this confuses the ambigator since `field == :hour` vs `nil` that the current AmPm does